stable/datadog/README.md

```diff
@@ -61,6 +61,8 @@ The following tables lists the configurable parameters of the Datadog chart and
 | `daemonset.tolerations`     | List of node taints to tolerate (requires Kubernetes >= 1.6) | `nil`           |
 | `daemonset.useHostNetwork`  | If true, use the host's network    | `nil`                                     |
 | `daemonset.useHostPort`     | If true, use the same ports for both host and container  | `nil`               |
+| `kube-state-metrics.rbac.create`| If true, create & use RBAC resources for kube-state-metrics | `true`       |
+| `kube-state-metricsrbac.rbac.serviceAccount` | existing ServiceAccount to use (ignored if rbac.create=true) for kube-state-metrics | `default` |
```

Comment:
`metricsrbac` should be `metrics`

Reply:
@hkaj Thank for your comment!
Sorry...I modiefied README in https://github.com/kubernetes/charts/pull/2969/commits/0c6823ff893c5a37476d0343ec4ef0539ca49afd